packages/apm/src/integrations/tracing.ts

```diff
@@ -382,15 +395,11 @@ export class Tracing implements Integration {
       // Gatekeeper if performance API not available
       return;
     }
+
     logger.log('[Tracing] Adding & adjusting spans using Performance API');
-    let navigationOffset = 0;
-    if (
-      transactionSpan.op === 'navigation' &&
-      transactionSpan.data &&
-      typeof transactionSpan.data.offset === 'number'
-    ) {
-      navigationOffset = transactionSpan.data.offset;
-    }
+
+    const timeOrigin = Tracing._msToSec(performance.timeOrigin);
```

Comment:
@rhcarvalho I came across this polyfill to replace an older Chrome API: https://developers.google.com/web/updates/2017/12/chrome-loadtimes-deprecated#startloadtime

wondering if we could make use of it?

Reply:
Yes.

Let's open the can of worms.
Need to make sure whether the times in entries are relative to `timeOrigin` or `navigationStart`, if the two ever differ -- @HazAT and I have only seen `performance.getEntriesByType('navigation')[0].startTime === 0`.

Reply:
This potential update could be something that can be in a new PR; not in this PR 👍

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/PerformanceEntry/startTime

> The `startTime` property returns the first recorded timestamp of the performance entry.
>
> The value returned by this property depends on the performance entry's type:
> ...
> **"navigation" - returns the timestamp with a value of "0".**

Reply:
https://github.com/w3c/navigation-timing/issues/43

Seems that the `timeOrigin` and `navigationStart` should be the same in practice.

In that case, [startloadtime](https://developers.google.com/web/updates/2017/12/chrome-loadtimes-deprecated#startloadtime) is unnecessarily verbose, giving us nothing more than what we have now? @dashed wdyt?

Reply:
@rhcarvalho that makes sense. I didn't realize `navigation` is essentially `0`. Thank you for investigating this further 👍